tools/bdist_all_wheels.py

```diff
@@ -34,14 +31,25 @@
 import setuptools
 
 setuptools.setup(
-   name="odoo-addons-oca-{repo}",
+   name="odoo{series}-addons-oca-{repo}",
    description="Meta package for OCA {repo} Odoo addons",
    version="{branch}.{date}",
    install_requires={install_requires},
 )
 """
 
 
+def branch_to_series(branch):
+    if os.environ.get('SETUPTOOLS_ODOO_LEGACY_MODE'):
+        return ''
+    if branch.startswith('8.0'):
+        return '8'
+    elif branch.startswith('9.0'):
+        return '9'
```

Comment:
For sure it will be in the future. There will be another PR for that. I'm wating on https://github.com/odoo/odoo/pull/13237 for 10.0.